src/rust/rust-proto/src/graplinc/grapl/api/plugin_registry/v1beta1_client.rs

```diff
@@ -95,10 +106,14 @@ impl PluginRegistryServiceClient {
         &mut self,
         request: native::DeployPluginRequest,
     ) -> Result<native::DeployPluginResponse, PluginRegistryServiceClientError> {
-        let response = self
+        let response = match self
```

Comment:
a bit less copy-paste verbose version can be seen here

```
    ) -> Result<native::RunGeneratorResponse, GeneratorServiceClientError> {
        let response = self
            .proto_client
            .run_generator(proto::RunGeneratorRequest::from(request))
            .await
            .map_err(Status::from)?;
```
src/rust/rust-proto/src/graplinc/grapl/api/plugin_sdk/generators/v1beta1/client.rs

Reply:
fixed